src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -89,10 +90,9 @@ public Integer extractSuffix() {
 
 	protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
-
 		for ( final String key : webData.getKeys() ) {
 			final String attributeValue = webData.getAsString( key );
-			if ( attributeValue != null ) {
+			if ( attributeValue != null && !isIdentifyingAttributesKey( key ) ) {
```

Comment:
Does that mean we exclude identifying attributes from the actual state?

Reply:
No, the attribute name "state" is a bit misleading there I guess. The method is used to retrieve the attributes for an element. So we just exclude already existing IdentifyingAttributes content.